CHANGELOG.md

```
@@ -1,3 +1,7 @@
+## 0.1.0
```

markdownlint-cli2 (0.22.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)